packages/node/src/indexer/api.service.ts

```diff
@@ -121,11 +115,14 @@ export class ApiService implements OnApplicationShutdown {
       value: 1,
     });
     this.patchedApi = patchedApi;
-    this.patchApi();
+    await this.patchApi();
     return this.patchedApi;
   }
 
-  private patchApi(registry?: Registry): void {
+  private async patchApi(
+    registry?: Registry,
+    blockHash?: BlockHash,
+  ): Promise<void> {
     if (registry) {
       Object.defineProperty(this.patchedApi, 'registry', {
```

Comment:
is this still needed? I remember with `api.at` it's not.

Reply:
Can you add a comment once confirmed? What's the original purpose we did this here? Is it for api.findCall / api.findError/ api.findEvent ?